package.json

```diff
@@ -228,7 +228,7 @@
   "private": true,
   "dependencies": {
     "@department-of-veterans-affairs/formation": "^6.8.0",
-    "@department-of-veterans-affairs/formation-react": "^4.10.3",
+    "@department-of-veterans-affairs/formation-react": "^5.0.0",
```

Comment:
Version bump to take advantage of `onHeaderClick` for the `<SortableTable />` component! 🙂